packages/local_auth/local_auth_ios/lib/types/auth_messages_ios.dart

```diff
@@ -54,14 +61,16 @@ class IOSAuthMessages extends AuthMessages {
           lockOut == other.lockOut &&
           goToSettingsButton == other.goToSettingsButton &&
           goToSettingsDescription == other.goToSettingsDescription &&
-          cancelButton == other.cancelButton;
+          cancelButton == other.cancelButton &&
+          localizedFallbackTitle == other.localizedFallbackTitle;
 
   @override
   int get hashCode =>
       lockOut.hashCode ^
       goToSettingsButton.hashCode ^
       goToSettingsDescription.hashCode ^
-      cancelButton.hashCode;
+      cancelButton.hashCode ^
+      localizedFallbackTitle.hashCode;
```

Comment:
just curious: i'm wondering does dart auto synthesize the hash function (like in swift)? manually implement this can be error prone (e.g. forgetting to include an attribute). 

Interesting choice using xor to combine hashes, because normally we use `aBigPrimeNumber * A + B`. Using xor comes with a few drawbacks like missing order information, common collision with `+x`, `-x` pairs, etc.

Reply:
for example, in java's [Arrays](https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/util/Arrays.java#l3948) we use 

```
  result = 31 * result + element;
```

Reply:
> Also interesting choice using xor to combine hashes

We should update this to the new `Object.hash`; that's out of scope for this PR though. I'll do a sweep for old manual hash implementations and send you a PR.